js-core/homeworks/homework-14/index.html

```diff
@@ -0,0 +1,97 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Home work 14</title>
+	<style>
+		*{
+			margin:0;
+			padding: 0;
+			font-size: 16px;
+			font-family: Arial;	
+			box-sizing: border-box;	
+		}
+		div.test-block{
+			margin: 20px 50px;
+		}
+		div.test-block p{
+			text-align: center;
+			margin-bottom: 40px;
+		}
+		form > ol{
+			padding-left: 25px;
+			margin-bottom: 25px;
+		}
+		form > ol > li {
+			margin-bottom: 25px;
+		}
+		form > ol > li > ul {
+			margin-top: 15px;
+		}
+		form > ol > li > ul > li{
+			list-style: none;
+			padding: 5px 0;
+		}
+		form input{
+			background-color: lightblue;
+		}
+		form input[type='submit']{
+			padding: 7px;
+		}
+	</style>
+</head>
+<body>
+
+	<script src="src/main.js"></script>
+
+
+<!-- 	<div class="test-block">
```

Comment:
please do not commit commented code it makes it hard to understand if it a mistake or commented solution or what do you really mean.

Your homework should be clear without commented code

Reply:
Deleted